server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -325,6 +326,34 @@ export class DeliLambdaFactory
 			});
 		});
 
+		deliLambda.on("noClient", () => {
+			const baseLumberjackProperties = getLumberBaseProperties(documentId, tenantId);
+			const handler = async () => {
+				// Set activity timer to reduce session grace period for ephemeral containers if cluster is in draining
+				if (document?.isEphemeralContainer && this.clusterDrainingChecker) {
+					const isClusterDraining = await this.clusterDrainingChecker.isClusterDraining();
+					if (isClusterDraining) {
+						Lumberjack.info(
+							"Cluster is under draining and NoClient event is received",
+							baseLumberjackProperties,
+						);
+						if (updateActivityTime) {
+							// Set session activity time to be 2 minutes later.
```

Comment:
Might we want to make this configurable?

Reply:
We use the default value (10 sec) and currently do not have the need to configure this value. Will make it configurable in the future if needed.